components/camel-jira/pom.xml

```diff
@@ -37,6 +37,7 @@
             org.apache.camel*;version=${camel.version}
         </camel.osgi.export>
         <camel.osgi.import>
+            com.google.common*;version="[33,34)",
```

Comment:
Old guava (31) was required, so had to enforce the version

Reply:
OK